src/Illuminate/Support/Collection.php

```diff
@@ -726,6 +726,24 @@ public function forPage($page, $perPage)
     }
 
     /**
+     * Returns an array with two elements. Items in the first element did pass
```

Comment:
Short descriptions must be "sort". Exactly one sentence and one line. The rest must be seperated.